### PR TITLE
Add ProActive in Kubernetes installation doc

### PR DESCRIPTION
**Pull request by mboussaa**

### CODE REVIEW

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 1]
```diff
@@ -498,7 +498,1246 @@ The user can access any of the four portals using the default credentials: `admi
 
 Your ProActive Scheduler is now running as a set of container services!
 
+==== How to install ProActive using Kubernetes
 
+ProActive could be easily deployed and started in a Kubernetes cluster. For this purpose, we provide a ready-to-use Kubernetes configuration files that enable you running ProActive Workflows and Scheduling as a set of Pods.
```
Comment (mklkun) [inline src/docs/admin/ProActiveAdminGuide.adoc:503]: a ready-to-use Kubernetes configuration files => ready-to-use Kubernetes configuration files  (delete `a`)

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 2]
```diff
@@ -498,7 +498,1246 @@ The user can access any of the four portals using the default credentials: `admi
...
+        secret:
+          secretName: cluster-config-2cd457dbmc
+
+----
+
+
+====== Set up ProActive Pods using MySQL Database
+
+The following configuration could be used to run the ProActive server Pod with an external MySQL Database Pod.
+It will start up to three Pods, one for the server, one for the Nodes, and one for the Database.
```
Comment (mklkun) [inline src/docs/admin/ProActiveAdminGuide.adoc:1296]: one for the server => one for the Server

**Review by mklkun — APPROVED** [review]
Two small typos.

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 3]
```diff
@@ -498,7 +498,1246 @@ The user can access any of the four portals using the default credentials: `admi
...
+
+====== Set up ProActive Pods using embedded HSQL Database
+The following YAML Kubernetes configuration could be used to run the ProActive server Pod with an embedded HSQL Database.
+It will start up to two Pods, one for the server and one for the Nodes.
+
+[source]
+----
+apiVersion: v1
+data:
+  DB_CATALOG_PASS: SjSWUF72s?C*eWhB
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc:524]: you should mention which variables in this section can be/should be edited.

As the file is public in the documentation, a customer you should edit passwords 

And I see many parameters below that should be modified, you could provide a table with
the name of the parameter, its description (and whether the parameter should be left as is or modified), the default value

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 4]
```diff
@@ -498,7 +498,1246 @@ The user can access any of the four portals using the default credentials: `admi
...
+The Kubernetes YAML configuration pulls the ProActive server, Node, and Database images from the https://dockerhub.activeeon.com/[Private Activeeon Docker Registry^].
+
+To be able to pull images from the https://dockerhub.activeeon.com/[Private Activeeon Docker Registry^], a ProActive enterprise licence and access credentials are required. Please contact us at contact@activeeon.com to request access.
+
+Here are the installation steps you need to follow in order to have ProActive running in a Kubernetes cluster:
+
+====== Set up ProActive Pods using embedded HSQL Database
+The following YAML Kubernetes configuration could be used to run the ProActive server Pod with an embedded HSQL Database.
+It will start up to two Pods, one for the server and one for the Nodes.
+
```
[the remark this answers is not in the document]
Comment (mboussaa) [reply src/docs/admin/ProActiveAdminGuide.adoc:519]: ok done

File: src/docs/admin/ProActiveAdminGuide.adoc (hunk 1 above)
Comment (mboussaa) [reply src/docs/admin/ProActiveAdminGuide.adoc]: ok done

File: src/docs/admin/ProActiveAdminGuide.adoc (hunk 2 above)
Comment (mboussaa) [reply src/docs/admin/ProActiveAdminGuide.adoc]: ok done, I used Server everywhere in the section

Comment (mklkun) [reply src/docs/admin/ProActiveAdminGuide.adoc]: Yes exactly. This is the reason of my comment 👍🏽

**Review by mklkun — APPROVED** [review]

**Comment by fviale** [comment]
Retest this please

**Review by fviale — APPROVED** [review]
LGTM

**Pull request merged**